specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2015-01-01/resourcehealth.json

```diff
@@ -351,6 +500,10 @@
               "description": "Timestamp for when the health was last checked. ",
               "format": "date-time"
             },
+            "isArmResource": {
```

Comment:
One thing to consider is if this would be more future-proof as enum, such as resourceType or resourceManager with values of ARM or other. If you may want to distinguish between types of non-ARM resources in the future, it's more trivial to expand a string enum.

Reply:
Thanks very much for your suggestion. This flag would just stay as boolean as it only indicate to the client that whether to show further details for child resource. If it is arm type resource, we would be able to get more details. If not, client/caller would not be able to get child health status. That's it. As of now, I don't see necessary of making any future-proof on this property. :)